docs/content/en/docs/packages/packagesecrmigration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Packages regional ECR migration"
+linkTitle: "Packages regional ECR migration"
+weight: 4
+description: >
+    Migrating packages ECR registry to packages registry for EKS Anywhere subscription
+---
+
+When an EKS Anywhere Enterprise Subscription is created, the AWS account that created the subscription is granted access to EKS Anywhere Curated Packages in the AWS Region where the subscription was created. If an existing EKS Anywhere cluster has curated packages access configured with the non-subscription based licens; we need to migrate the packages to consume the regional ECR registries.
```

Comment:
I think we should say,
"When you purchase an EKS Anywhere Enterprise Subscription through the Amazon EKS console or APIs, the AWS account that purchased the subscription is automatically granted access to EKS Anywhere Curated Packages in the AWS Region where the subscription is created. If you received trial access to EKS Anywhere Curated Packages or if you have an EKS Anywhere Enterprise Subscription that was created before October 2023, then you need to migrate your EKS Anywhere Curated Packages configuration to use the latest ECR regional repositories."